emails/models.py

```diff
@@ -70,14 +70,12 @@ def valid_available_subdomain(subdomain, *args, **kwargs):
 
 
 def default_server_storage():
-    return datetime.now(timezone.utc) > settings.PREMIUM_RELEASE_DATE
+    return True
```

Comment:
I kept this since a migration (specifically `0029_profile_add_deleted_metric_and_changeserver_storage_default.py` for the `emails` app) pointed to this function as the one to generate a default value. Not sure if there's a better way to do that.

Reply:
Good idea: a12cf6be